pkg/lib/cockpit-components-empty-state.jsx

```diff
@@ -24,7 +24,7 @@ import { EmptyStateActions, EmptyState, EmptyStateBody, EmptyStateFooter, EmptyS
 import { Spinner } from "@patternfly/react-core/dist/esm/components/Spinner/index.js";
 import "./cockpit-components-empty-state.css";
 
-export const EmptyStatePanel = ({ title, paragraph, loading, icon, action, isActionInProgress = false, onAction, secondary, headingLevel = "h1" }) => {
+export const EmptyStatePanel = ({ title, paragraph, loading, icon, action, isActionInProgress = false, onAction, actionVariant = "link", secondary, headingLevel = "h1" }) => {
```

Comment:
Is the "most of the time" based on actual counting or gut feeling? We use this a lot, also in podman/machines, and I feel the other way around. I.e. keep the "primary" default, and make it link or secondary etc. in the new places where we need it?

Reply:
This is indeed a backwards incompatible change, this whole panel is abused or used wrong :)

```
                        <EmptyStatePanel title={ cockpit.format(_("VM $0 does not exist on $1 connection"), cockpit.location.options.name, cockpit.location.options.connection) }
                                         action={
                                             <Button variant="link"
                                                     onClick={() => cockpit.location.go(["vms"])}>
                                                 {_("Go to VMs list")}
                                             </Button>
                                         }
```

So Garrett's two cents where, most of the time you want a link, but I am also fine to make this non-breaking, that is better imo.